### ISSUE TITLE
Feature Request - Convert .jxr to .exr

### ISSUE THREAD

**Issue by redrockvp** [brion/hdrfix#9]
Hi there,

Great tool you've made here :) Still tweaking settings to play around with HDR screenshots from Game Bar, but working so far. 

Wanted to ask if it is at all feasible to convert/save .jxr files as .exr files? The .jxr file plugin for Photoshop is pretty broken and I'd love to just edit the HDR images directly and then export to sRGB/do tonemapping as the last step, rather than the first step.

**Comment by brion** [comment]
Yes, that's likely relatively simple (in that the heavy lifting of reading the file format should exist as a usable open source library ready to use :) ), and I hope to get that in the next time I get some time on this project. It's hot this summer, so I've got less flight simulator time but can work on code in the air conditioning... ;)

**Comment by redrockvp** [comment]
That would be amazing :D I'm very keen to see what I can do in Photoshop with the raw HDR image. I primarily do car photography in the Forza series currently and the HDR implementation in the later titles is really good.
